examples/integrations/detectron2/coco_object.ipynb

```diff
@@ -460,7 +460,7 @@
     {
       "metadata": {},
       "source": [
-        "pairs = get_label_pairs(val_labels, labels_mea, filter_mismatch = True)\n",
+        "pairs = lb.get_label_pairs(val_labels, labels_mea, filter_mismatch = True)\n",
```

Comment:
Functions related to metrics are not included in the `lb` namespace since this has mostly been replaced by autometrics. 

This is how it can be imported instead:
```
from labelbox.data.metrics.group import get_label_pairs
```

You can also leave out this change, I can fix this for the rest of `get_label_pairs` usages in notebooks.

Reply:
thanks, @whistler . I will do the import you suggest... o.w. the notebooks it broken (I will test it in python, since this notebook does not run to my knowlesge)